proto/tendermint/abci/types.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+package tendermint.abci;
+
+option go_package = "github.com/lazyledger/lazyledger-core/abci/types";
+
+// For more information on gogo.proto, see:
+// https://github.com/gogo/protobuf/blob/master/extensions.md
+import "google/protobuf/timestamp.proto";
+import "gogoproto/gogo.proto";
+
+// This file is copied from http://github.com/tendermint/abci
+// NOTE: When using custom types, mind the warnings.
+// https://github.com/gogo/protobuf/blob/master/custom_types.md#warnings-and-issues
+
+// Validator
+message Validator {
+  bytes address = 1;  // The first 20 bytes of SHA256(public key)
+  // PubKey pub_key = 2 [(gogoproto.nullable)=false];
+  int64 power = 3;  // The voting power
+}
```

Comment:
We might want to change that to Aggregator. But we can think about this further and do this in a followup PR.